features/step_definitions/schools/dfe_signin_steps.rb

```diff
@@ -1,4 +1,8 @@
+include DFESignInAPI
+
 Given("I am logged in as a DfE user") do
+  stub_dfe_sign_in_organisations
```

Comment:
I'm a little confused by this. My reading of the code is this wont work when cucumber is run against a separate server? but obviously the test suite is passing.

Reply:
Can you either verify that it is behaving as expected, or add an explanation in a note. Thanks

Reply:
Yep, pulled this out